quadproduction/OpenPype
# Enhancement: Set Deadline prio pool limit machine limit plugin per task type

## Changelog Description
Create new settings for priority, pools, machine limit and plugin limit filtered by task type and host, replacing these in the host submitters settings. Also add these options in the publisher for Maya and Nuke.

## Testing notes:
1. set a profile in the settings
2. create an instance that can be sent to Deadline in Maya or Nuke
3. verify that the data in the publisher are correct and publish
4. verify that the data in Deadline are correct too
